mep/people/views.py

```diff
@@ -54,14 +54,33 @@ def get_form_kwargs(self):
         kwargs['data'] = form_data
         return kwargs
 
-
     def get_form(self, *args, **kwargs):
         # initialize the form, caching on current instance
         if not self._form:
             self._form = super().get_form(*args, **kwargs)
+        # set minimum and maximum years for date range field
+            min_max = self.get_year_range()
+            if min_max:
+                self._form.set_membership_dates_placeholder(*min_max)
         # Get facets from solr return
         return self._form
 
+    @staticmethod
+    def get_year_range():
+        """Return the earliest and latest years for any account activity in
+        the library.
+
+        :return: Min and max years as integers or None
+        :rtype: tuple or None
+        """
+
+        stats = PersonSolrQuerySet().stats('account_start', 'account_end')\
```

Comment:
I didn't catch this on my quick review yesterday: you should use `account_years` for both max and min rather than using start and end. In fact, I'm wondering now if we even need start and end indexed separately - I think we don't, but we don't need to fix it right now.